src/lsp/cobol_parser/text_tokenizer.ml

```diff
@@ -735,7 +738,7 @@ let enable_intrinsics state token tokens =
     let tokens = retokenize_after Enabled_intrinsics state tokens in
     let token, tokens = List.hd tokens, List.tl tokens in
     if show `Tks state then
-      Pretty.error "Tks': %a@." pp_tokens tokens;
+      Pretty.error "Tks': %a@." pp_token'_list tokens;
```

Comment:
Please keep `type tokens = … with_loc list` and `pp_tokens`, that's shorter and perfectly fine for a module internal.